modules/request/index.js

```diff
@@ -40,9 +42,14 @@ Request.prototype.validate = function () {
   if (this.data()) {
     validators = validators.concat([verifyContentType, verifyDataObject]);
   }
+
+  // does this.validators needs a better name? controllerValidator, userValidators?
+  validators = validators.concat(this.validators);
+
   var endpoint = {
```

Comment:
Let's not make this object... instead, let's pass in the entire config and let the validator methods find the id and `source.typeName()` on their own.